src/Microsoft.AspNet.Razor.Runtime/ITextWriterCopyable.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Contains the methods necessary to be implemented copy a particular
+    /// type to a <see cref="TextWriter"/>.
+    /// </summary>
+    public interface ITextWriterCopyable
+    {
+        /// <summary>
+        /// Method which needs to be implemented to copy the particular type of a <see cref="TextWriter"/>.
+        /// </summary>
+        /// <param name="writer">The <see cref="TextWriter"/> to which the type must be copied to.</param>
+        void CopyToWriter(TextWriter writer);
```

Comment:
I made it sepecific because I dint want it to overlap with `CopyTo(TagHelperContent)` in `DefaultTagHelperContent` implementation. And moreover, this implies the intent more clearly right?